RecoTracker/Configuration/test/reTrackingCosmics_cfg.py

```diff
@@ -4,16 +4,16 @@
 process.load("FWCore.MessageLogger.MessageLogger_cfi")
 process.load("CondCore.DBCommon.CondDBSetup_cfi")
 process.load("Configuration.StandardSequences.MagneticField_38T_cff")
-process.load("Configuration.StandardSequences.Geometry_cff")
+process.load("Configuration.StandardSequences.GeometryDB_cff")
 process.load("Configuration.StandardSequences.ReconstructionCosmics_cff")
 process.load("Configuration.EventContent.EventContentCosmics_cff")
 
 # Conditions (Global Tag is used here):
 process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
 ########## include here the correct global tag. ########## 
 # Have a look here: https://twiki.cern.ch/twiki/bin/view/CMS/SWGuideFrontierConditions
-# to check if there are newer tags
-process.GlobalTag.globaltag = "CRAFT_ALL_V11::All"
+# 2022
+process.GlobalTag.globaltag = 'auto:phase1_2022_realistic'
```

Comment:
the example below implies that this is data. This GT seems wrong, or at least inconsistent with the inputs.

Reply:
Yes, for data I suggest to use
```
process.GlobalTag.globaltag = 'auto:run3_data_prompt`
```

Reply:
The input file is from 2008, likely it does not exist anymore, but certainly the run3 GT is lacking by construction very old IoVs

Reply:
I think [this ](https://github.com/cms-sw/cmssw/pull/38551#discussion_r915031541) has not been taken into account yet, despite the alca signature.

Reply:
Marco, I saw your comments, but what is it that you are suggesting? Should we just delete this file? If it was not used since 2008... My point is, now the cff config to import exists, the GT exists... the person if anybody will want to use this file will modify the input to something Run-3 and then it will work. I dont think it's @jeongeun responsibility to do this. But I do think that making partial repairs to the file has value as well. Or we really should just delete the file.

Reply:
there's plenty of Run3 cosmics data now.